mkat_fpga_tests/test_cbf.py

```diff
@@ -487,10 +492,52 @@ def _delays_setup(self):
         baseline_lookup = get_baselines_lookup(initial_dump)
         # Choose baseline for phase comparison
         baseline_index = baseline_lookup[('m000_x', 'm000_y')]
+
+        sync_time = initial_dump['sync_time']
+        scale_factor_timestamp = initial_dump['scale_factor_timestamp']
+        time_stamp = initial_dump['timestamp']
+        # To do(MM) 2015-10-07, get int time from dump
+        # (int_time = initial_dump['int_time'])
+        int_time = self.xengops.get_acc_time()
+        dump_1_timestamp = (sync_time + time_stamp/scale_factor_timestamp)
+        t_apply = dump_1_timestamp + 5*int_time
+
         return {
             'baseline_index': baseline_index,
             'baseline_lookup': baseline_lookup,
-            'initial_dump': initial_dump}
+            'initial_dump': initial_dump,
+            'sync_time':sync_time,
+            'scale_factor_timestamp':scale_factor_timestamp,
+            'time_stamp':time_stamp,
+            'int_time':int_time,
+            'dump_1_timestamp':dump_1_timestamp,
+            't_apply' : t_apply
+            }
+
+    def _plot_and_save(self, freqs, actual_data, expected_data, plot_filename,
```

Comment:
This function is probably specific to phases or delays. Perhaps rename it something like _plot_phase_results()? Perhaps leave off the _and_save_ bit since we probably know its doing that :)